File: share/docker-build/amd64/go/README.go
package main
//go run README.go

import "fmt"

func main() {
	var (
		country, ProxySite string
	)

	country = "China"
	ProxySite = "https://goproxy.cn"
	fmt.Printf("If you live in\x1b[%dm %v. \x1b[0m\n", 32, country)
	fmt.Printf("Then you can run the following\x1b[%dm commands. \x1b[0m\n", 31)

	fmt.Println("go env -w GO111MODULE=on")
	fmt.Printf("go env -w GOPROXY=%v,direct\n", ProxySite)
}

File: share/docker-build/arm64/node/README.js
//node README.js
var country = "China";
console.log('\x1B[36m%s\x1B[0m \x1B[31m%s\x1B[0m.', "If you live in", country);
console.log("%s \x1B[32m%s\x1B[0m.","Then you can run the following", "commands");
console.log("\x1B[33m%s\n%s\n%s\n\x1B[0m", "npm config set registry https://repo.huaweicloud.com/repository/npm/", "npm config set disturl https://repo.huaweicloud.com/nodejs", "npm config set electron_mirror https://repo.huaweicloud.com/electron/");
